samples/net/cloud/aws_mqtt/prj.conf

```diff
@@ -0,0 +1,70 @@
+CONFIG_NETWORKING=y
+
+# Disable IPv6 support
+CONFIG_NET_IPV6=n
+
+# Enable IPv4 support
+CONFIG_NET_IPV4=y
+CONFIG_NET_IF_MAX_IPV4_COUNT=2
+CONFIG_NET_IF_UNICAST_IPV4_ADDR_COUNT=3
+
+# Enable TCP support
+CONFIG_NET_TCP=y
+
+# Enable Sockets support
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+
+# Enable DHCPv4 support
+CONFIG_NET_DHCPV4=y
+
+# Enable SOCKS5 proxy support
+CONFIG_SOCKS=n
+
+# Enable MQTT Lib support
+CONFIG_MQTT_LIB=y
+CONFIG_MQTT_LIB_TLS=y
+
+# Network configuration
+CONFIG_NET_CONFIG_SETTINGS=y
+
+# Network connection manager
+CONFIG_NET_CONNECTION_MANAGER=y
+
+CONFIG_NET_MGMT_EVENT_STACK_SIZE=1024
+CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=5
+CONFIG_NET_MGMT_EVENT_LOG_LEVEL_DBG=n
+
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_SERVER_IP_ADDRESSES=y
+CONFIG_DNS_SERVER1="8.8.8.8"
+CONFIG_DNS_RESOLVER_ADDITIONAL_BUF_CTR=2
+CONFIG_NEWLIB_LIBC=y
+
+CONFIG_INIT_STACKS=y
+CONFIG_NET_SHELL=y
+
+CONFIG_MAIN_STACK_SIZE=4096
+
+# Enable Logging support
+CONFIG_LOG_IMMEDIATE=y
+CONFIG_NET_LOG=y
+CONFIG_MQTT_LOG_LEVEL_DBG=y
+CONFIG_SOCKS_LOG_LEVEL_DBG=n
+CONFIG_NET_CONNECTION_MANAGER_LOG_LEVEL_DBG=n
+CONFIG_DNS_RESOLVER_LOG_LEVEL_DBG=n
+CONFIG_NET_CONTEXT_LOG_LEVEL_DBG=n
+CONFIG_SOC_PART_NUMBER_SAME70Q21B=y
```

Comment:
Remove board specific stuff from this file. If you need to enable something board specific, create `boards` directory and put board file there (see example in various networking samples what this means).